### PR TITLE
Don't apply the "objectcreation" completion tag

### PR DESCRIPTION
**Pull request by rchande**
This prevents item unification

### CODE REVIEW

**Comment by rchande** [comment]
Tag @dotnet/roslyn-ide @mattwar for review

**Comment by CyrusNajmabadi** [comment]
👍

File: src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb [hunk 1]
```diff
@@ -2801,6 +2801,27 @@ End Class]]></Document>)
             End Using
         End Function
 
+        <WorkItem(15011, "https://github.com/dotnet/roslyn/issues/15011")>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function SymbolAndObejctPreselectionUnification() As Task
```
Comment (Pilchie) [inline src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb]: Spelling - `Obejct`

File: src/Features/Core/Portable/Completion/Providers/AbstractObjectCreationCompletionProvider.cs [hunk 2]
```diff
@@ -23,8 +23,6 @@ internal abstract class AbstractObjectCreationCompletionProvider : AbstractSymbo
         /// </summary>
         protected abstract SyntaxNode GetObjectCreationNewExpression(SyntaxTree tree, int position, CancellationToken cancellationToken);
 
-        private static readonly ImmutableArray<string> s_Tags = ImmutableArray.Create(CompletionTags.ObjectCreation);
```
Comment (Pilchie) [inline src/Features/Core/Portable/Completion/Providers/AbstractObjectCreationCompletionProvider.cs]: Should this tag be removed?  What is it used for elsewhere?

Comment (rchande) [inline src/Features/Core/Portable/Completion/Providers/AbstractObjectCreationCompletionProvider.cs]: Not used for anything. Removed.

File: src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb [hunk 3]
```diff
@@ -2801,6 +2801,27 @@ End Class]]></Document>)
...
+
+End Module
+]]></Document>)
+
+                state.SendInvokeCompletionList()
+                Await state.WaitForAsynchronousOperationsAsync()
+                Dim psi = state.CurrentCompletionPresenterSession.CompletionItems.Where(Function(i) i.DisplayText.Contains("ProcessStartInfo")).ToArray()
+                Assert.Equal(1, psi.Length)
+            End Using
+        End Function
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb]: Can we have a C# test?

**Pull request merged**